src/platform/cordova.scss

```diff
@@ -44,9 +44,6 @@ $cordova-statusbar-padding-modal-max-width:         767px !default;
 
   > .toolbar.statusbar-padding:first-child {
     padding-top: calc(#{$cordova-statusbar-padding} + #{$toolbar-padding});
-
-    height: calc(#{$toolbar-height} + #{$cordova-statusbar-padding});
-    min-height: calc(#{$toolbar-height} + #{$cordova-statusbar-padding});
```

Comment:
But there is always a min-height set in 
.toolbar-ios, .toolbar-md and .toolbar-wp :-)

```
.toolbar-ios {
    padding: 1rem;
    min-height: 44px;
}
```